src/mlpack/methods/ann/rbm/rbm.hpp

```diff
@@ -87,10 +87,11 @@ class RBM
    * parameters vector directly with Parameters() and modify it as desired.
    *
    * @param optimizer Optimizer type.
+   * @param callbacks Callback Functions
    * @return The final objective of the trained model (NaN or Inf on error).
    */
-  template<typename OptimizerType>
-  double Train(OptimizerType& optimizer);
+  template<typename OptimizerType, typename... CallbackType>
```

Comment:
Can you add a description for the ``OptimizerType`` and ``CallbackType`` type, as done here:

https://github.com/mlpack/mlpack/blob/2560dd21db93e4c0d72c9dc094b8c041112a15bd/src/mlpack/methods/ann/gan/gan.hpp#L115-L116